java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/VirtualObservableReaderImpl.java

```diff
@@ -28,6 +32,13 @@
 
     private static final Logger logger = LoggerFactory.getLogger(VirtualObservableReaderImpl.class);
```

Comment:
the class VirtualObservableReaderImpl should maybe extend VirtualReaderImpl instead of AbstractReader